docs/SetupGuide_Java.md

```diff
@@ -42,6 +42,14 @@ These instructions will guide you through creating your Function App and adding
         "version": "[4.*, 5.0.0)"
     }
     ```
+    Add the Java library for SQL bindings to the pom.xml file.
+    ```xml
+    <dependency>
+        <groupId>com.microsoft.azure.functions</groupId>
+        <artifactId>azure-functions-java-library-sql</artifactId>
+        <version>0.1.1</version>
```

Comment:
How will we keep this version up to date? Is this documented in release instructions?

Alternatively, can we use wildcards to avoid putting an explicit version here?

Reply:
Good point - I can add this to release instructions (we already have a note to update Microsoft docs). The library itself should not be updated very often.

Reply:
Updated to `[0.1.1,)` so that it can pick up new versions.